### PR TITLE
Update to match doc

### PR DESCRIPTION
**Pull request by blegat**

### CODE REVIEW

File: src/attributes.jl [hunk 1]
```diff
@@ -162,7 +162,7 @@ canset(instance, ConstraintPrimal(), ConstraintIndex{VectorAffineFunction{Float6
 """
 function canset end
 canset(instance::AbstractInstance, attr::AnyAttribute) = false
-canset(instance::AbstractInstance, attr::AnyAttribute, ref::Index) = false
+canset(instance::AbstractInstance, attr::AnyAttribute, ref::Type{<:Index}) = false
 canset(instance::AbstractInstance, attr::AnyAttribute, refs::Vector{<:Index}) = false
```
Comment (mlubin) [inline src/attributes.jl]: Should this vector version still exist?

Comment (blegat) [reply src/attributes.jl]: I'd vote to remove it

Comment (mlubin) [reply src/attributes.jl]: Is it even documented?

Comment (mlubin) [reply src/attributes.jl]: Yes, it is. But anyway it's redundant. Let's remove it.

**Pull request merged**